doc/classes/@GDScript.xml

```diff
@@ -338,6 +364,26 @@
 			<argument index="1" name="y" type="float">
 			</argument>
 			<description>
+				Returns the floating-point remainder of x/y that wraps equally in positive and negative.
```

Comment:
A bit confusing. It's supposed to be a modulo according to the current docs.
`Returns the remainder of 'x' / 'y'. If x is negative, returns the remainder of '-x' / 'y'` ? Not sure I got exactly how the method works - I don't even know when you'd use it.

Reply:
I agree its a bit confusing. I'm not sure where I would use it.  Thats why I included the lengthy example.